charts/calibre-web/values.yaml

```diff
@@ -1,136 +1,41 @@
-# Default values for calibre-web.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+# Default values for Calibre-Web.
 
 image:
   repository: linuxserver/calibre-web
-  tag: 0.6.8-ls74
   pullPolicy: IfNotPresent
+  tag: version-0.6.9
 
-# upgrade strategy type (e.g. Recreate or RollingUpdate)
-strategyType: Recreate
-
-# Probes configuration
-probes:
-  liveness:
-    initialDelaySeconds: 60
-    failureThreshold: 5
-    timeoutSeconds: 10
-  readiness:
-    initialDelaySeconds: 60
-    failureThreshold: 5
-    timeoutSeconds: 10
-
-nameOverride: ""
-fullnameOverride: ""
-
-# Linuxserver.io additional layers.
-# Enables additional features for the image, at the cost of increased size and
-# possible incompatabilities with certain architectures. Disabled by default.
-dockerMods:
```

Comment:
Should we maybe somehow refer to the LSIO page for the available configuration options (env vars)?

Reply:
Bad copy paste, will add back .helmignore